MantidQt/CustomInterfaces/src/Reflectometry/ReflMainWindowPresenter.cpp

```diff
@@ -0,0 +1,130 @@
+#include "MantidQtCustomInterfaces/Reflectometry/ReflMainWindowPresenter.h"
+#include "MantidQtCustomInterfaces/Reflectometry/IReflMainWindowView.h"
+#include "MantidQtCustomInterfaces/Reflectometry/IReflRunsTabPresenter.h"
+#include "MantidQtCustomInterfaces/Reflectometry/IReflSettingsTabPresenter.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+/** Constructor
+* @param view :: [input] The view we are managing
+* @param runsPresenter :: [input] A pointer to the 'Runs' tab presenter
+* @param settingsPresenter :: [input] A pointer to the 'Settings' tab presenter
+*/
+ReflMainWindowPresenter::ReflMainWindowPresenter(
+    IReflMainWindowView *view, IReflRunsTabPresenter *runsPresenter,
+    IReflSettingsTabPresenter *settingsPresenter)
+    : m_view(view), m_runsPresenter(runsPresenter),
+      m_settingsPresenter(settingsPresenter) {
+
```

Comment:
One alternative approach is to get the `IReflMainViewWindow` to return to you all sub presenters:

```cpp
view->getRunsPresenter();
view->getSettingsPresenter();
```
I think this form is slightly more testable, but I don't have a strong opinion.